packages/html-spec/src/notification/notification.json

```diff
@@ -0,0 +1,103 @@
+{
+    "$schema": "../../schemas/component.schema.json",
+    "title": "Kendo Notification HTML Spec",
+    "component": "notification",
+    "selector": [
+        ".k-notification"
+    ],
+    "options": [
+        {
+            "name": "themeColor",
+            "values": [
+                {
+                    "name": "base",
```

Comment:
We already have the `k-notification` class on the root element.
If the `base` option does not add a specific class it will probably be better if the whole `base` element is removed from the `values` array.